### PR TITLE
Game Master Update 0.103.3

### PR DESCRIPTION
**Pull request by dandesousa**

### CODE REVIEW

File: versions/latest/GAME_MASTER.json [hunk 1]
```diff
@@ -30972,6 +30989,23 @@
       "buddyOffsetFemale": [10.0, 0.0, -80.0],
       "buddyScale": 19.0
     }
+  }, {
+    "templateId": "V0298_MOVE_BLAST_BURN",
+    "moveSettings": {
+      "movementId": 298,
```
Comment (celandro) [inline versions/latest/GAME_MASTER.json]: This needs to be fixed by updating the java proto package

Comment (dandesousa) [reply versions/latest/GAME_MASTER.json]: Thanks, I see that now -- I noticed its at the latest package available.

Going to be busy for the next few days but if someone doesn't beat me to it, I'll try and take a look and rebuilding the dependencies next week.

Comment (dominicbarnes) [reply versions/latest/GAME_MASTER.json]: (and I'm definitely not a java dev by any stretch, so I'm way out of the loop)

**Comment by dandesousa** [comment]
duplicate of #3 and #2

**Pull request closed without merging**